src/SimpleAzure.Storage.HybridQueue/HybridQueue.cs

```diff
@@ -186,73 +157,65 @@ public async Task DeleteMessageAsync<T>(HybridMessage<T> hybridMessage, Cancella
 
         _logger.LogDebug("About to receive queue message.");
 
-        var response = await _queueClient.ReceiveMessagesAsync(maxMessages, visibilityTimeout, cancellationToken);
-
-        if (response == null ||
-            response.Value == null)
+        var response = await _queueClient.ReceiveMessagesAsync(maxMessages, visibilityTimeout, cancellationToken).ConfigureAwait(false);
+        if (response?.Value is not { } messages)
         {
             _logger.LogDebug("Response was null or there were no Queue messages retrieved.");
-
-            return Enumerable.Empty<HybridMessage<T>>();
+            return Array.Empty<HybridMessage<T>>();
         }
 
-        _logger.LogDebug("Received {} messages from queue.", response.Value.Length);
-
-        var hybridMessageTasks = response.Value.Select(x => ParseMessageAsync<T>(x, cancellationToken));
+        _logger.LogDebug("Received {} messages from queue.", messages.Length);
 
-        var hybridMessages = await Task.WhenAll(hybridMessageTasks);
+        var hybridMessageTasks = messages.Select(x => ParseMessageAsync<T>(x, cancellationToken));
 
-        return hybridMessages;
+        return await Task.WhenAll(hybridMessageTasks).ConfigureAwait(false);

```

Comment:
I've been preferring having two lines here instead of one because when I need to debug something, I find it easier to read the result _before_ it gets returned.

I'm sure this could be because I'm not using my Tools, properly, though.

Reply:
In VS the debugger will show the return value of a method as you step out, or you can out a breakpoint on the closing brace, I think. I understand though, the codebase I inherited did this too, so I'm used to seeing it.

Reply:
Rejected. Personal pref.